clients/extension/src/background/index.ts

```diff
@@ -408,12 +408,13 @@ const handleRequest = (
       case RequestMethod.VULTISIG.SEND_TRANSACTION: {
         const [_transaction] = params
         getStandardTransactionDetails(
-          _transaction as TransactionType.WalletTransaction,
+          {
+            ..._transaction,
+            txType: _transaction.txType ?? 'Vultisig',
```

Comment:
can we use a strong type for `Vultisig` , `MetaMask`,`Ctrl` etc? 
you could address it in another PR